website/docs/cli/install/apt.html.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: "downloads"
+page_title: "APT Packages for Debian and Ubuntu"
+sidebar_current: "docs-cli-install-apt"
+description: |-
+  The HashiCorp APT repositories contain distribution-specific Terraform packages for both Debian and Ubuntu systems.
+---
+
+# APT Packages for Debian and Ubuntu
+
+The primary distribution packages for Terraform are `.zip` archives containing
+single executable files that you can extract anywhere on your system. However,
+for easier integration with configuration management tools and other systematic
+system configuration strategies, we also offer package repositories for
+Debian and Ubuntu systems, which allow you to install Terraform using the
+`apt install` command or any other APT frontend.
+
+If you are instead using Red Hat Enterprise Linux, CentOS, or Fedora, you
+might prefer to [install Terraform from our Yum repositories](yum.html).
+
+-> **Note:** The APT repositories discussed on this page are generic HashiCorp
+repositories that contain packages for a variety of different HashiCorp
+products, rather than just Terraform. Adding these repositories to your
+system will, by default, therefore make a number of other non-Terraform
+packages available for installation. That might then mask some packages that
+are available for some HashiCorp products in the main Debian and Ubuntu
+package repositories.
+
+## Repository Configuration
+
+The Terraform packages are signed using a private key controlled by HashiCorp,
+so in most situations the first step would be to configure your system to trust
+that HashiCorp key for package authentication. For example:
+
+```bash
+curl -fsSL https://apt.releases.hashicorp.com/gpg | sudo apt-key add -
+```
+
+After registering the key, you can add the official HashiCorp repository to
+your system:
+
+```bash
+sudo apt-add-repository "deb [arch=$(dpkg --print-architecture)] https://apt.releases.hashicorp.com $(lsb_release -cs) main"
+```
+
+The above command line uses the following sub-shell commands:
+
+* `dpkg --print-architecture` to determine your system's primary APT
+  architecture/ABI, such as `amd64`.
+* `lsb_release -cs` to find the distribution release codename for your current
+  system, such as `buster`, `groovy`, or `sid`.
+
+`apt-add-repository` usually automatically runs `apt update` as part of its
+work in order to fetch the new package indices, but if it does not then you
+will need to so manually before the packages will be available.
+
+To install Terraform from the new repository:
+
+```bash
+sudo apt install terraform
+```
+
+## Supported Architectures
+
+The HashiCorp APT server currently has packages only for the `amd64`
+architecture, which is also sometimes known as `x86_64`.
+
+There are no official packages available for other architectures, such as
+`arm64`. If you wish to use Terraform on a non-`amd64` system, download a
```

Comment:
I think a link to our downloads page is appropriate here, since you are directly referencing downloads.

Reply:
I'd decided not to link to the downloads page because I'm expecting it to _not_ be an appropriate link for "just download the `.zip` file" in future once the Downloads page revamp is done and it ends up being more like an overview of all of the options, including short instructions for these Linux distribution packages.

On reflection though, I wrote these pages up the way I did specifically _because_ that Downloads page redesign isn't expected to arrive in the very near future, so I will change these to link to the downloads page for now and then we can adjust to perhaps link to another page in this same directory (`zip.html.md`? :man_shrugging:) if the Downloads page starts being less directly about the `.zip` packages in the future.